translations/en.json

```diff
@@ -141,5 +141,10 @@
     "name": "City of Utrecht",
     "homepage-label": "homepage",
     "logo-label": "Coat of arms of the city of Utrecht"
+  },
```

Comment:
Dan moeten we even kijken hoe we dat met meertaligheid gaan doen; ik denk dat de beste optie gewoon met een hook zoals useLocale is en dat de beheerder dan in het configuratie object de labels per taal moet doorgeven. Bijv: 

```{ "en": "English label", "nl": "Nederlands label" }```